### DISCUSSION TITLE
Announcement of absence

### DISCUSSION THREAD

**Discussion by maggu2810** [openhab/openhab-core#3497] [Core Maintainers Discussions]
I am on holiday from next week until the beginning of April.
On these days I will not be available to review or comment anything.

Will be great if we can get some stuff merged this week, otherwise I cannot reply and it will stay open two additional weeks.

**Comment by cweitkamp** [comment]
Happy holidays.

> Will be great if we can get some stuff merged this week

Do you have some specific PRs in your mind? Do you like to give us a hint?

**Comment by maggu2810** [comment]
No special one, just mine non WIP PRs that do not really need conceptual discussions.

**Comment by maggu2810** [comment]
I had a look for the open ones.
https://github.com/openhab/openhab-core/pull/640 would make sense.
Then we can bump the Bnd version in the Maven projects (IDE plugin repo already contains 4.2.0).
